### PR TITLE
chore(docs): Document Cypress test configuration

### PR DESCRIPTION
**Pull request by bheiskell**

### CODE REVIEW

File: cypress/README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+1. Navigate to the repository > Settings > Environments
+2. Click "New environment"
+3. Name it exactly `requires-approval` and click "Configure environment"
+4. Check "Requires reviewers" and specify who is allowed to approve
+5. Click "Save protection rules"
+
+Note: No environment variables need to be configured here. They're inherited from the repository global variables.
+
+### 3. Actions permissions (Optional)
```
Comment (wrycu) [inline cypress/README.md:103]: It's not clear to me what this section does or why it's optional.

File: cypress/README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+Once Cypress is open:
+
+1. Add the project
+2. Select "E2E Testing"
+3. Select "Start E2E Testing in Chrome" (or the browser of your choice), which launches the browser
+4. Select the test to run (`*.cy.js` file) - :warning: the test is immediately run
+
+### Configuration
+
+The default Cypress configuration can be customized with a `cypress.env.json`. For example, to change the instance of FoundryVTT being tested to another port:
```
Comment (wrycu) [inline cypress/README.md:42]: Where do you create this file?

File: cypress/README.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+```shell
+# Install dependencies
+npm install
+
+# Run Cypress
+npx cypress run
+```
+
+## Developing locally
```
Comment (wrycu) [inline cypress/README.md:25]: Developing... tests locally?

File: cypress/README.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+{
+  "baseUrl": "http://localhost:30001"
+}
+```
+
+### Docker Compose
+
+To test FoundryVTT with Docker, use the `felddy/foundryvtt` docker image. This is what our GitHub Actions use.
+
+TODO: Add `docker-compose.yaml` example.
```
Comment (wrycu) [inline cypress/README.md:54]: Oh, look. "yaml" and not "yml". How very _correct_.

File: cypress/README.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+1. Navigate to the repository > Settings > Secrets and variables > Actions
+2. Repeat following for `FOUNDRY_LICENSE_KEY`, `FOUNDRY_PASSWORD`, and `FOUNDRY_USERNAME`:
+   1. Click "New repository secret"
+   2. Populate the secret name and its value
+   3. Click "Add secret"
+
+### 2. Environment
+
+Configure the `requires-approval` environment to require specific approvers to run the Cypress tests on a fork.
```
Comment (wrycu) [inline cypress/README.md:93]: It's more "from" a fork than "on" a fork, isn't it?

File: cypress/README.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+1. Navigate to the repository > Settings > Environments
+2. Click "New environment"
+3. Name it exactly `requires-approval` and click "Configure environment"
+4. Check "Requires reviewers" and specify who is allowed to approve
```
Comment (wrycu) [inline cypress/README.md:98]: Eh? So does it not automatically sync to repo contributors?

**Review by wrycu — COMMENTED** [review]
Left some comments. Some are clarification, some are hinting at improvements that should be made to wording.

File: cypress/README.md (hunk 6 above)
Comment (bheiskell) [reply cypress/README.md:98]: No, it doesn't. There might be a way to use a group, but I don't know how.

File: cypress/README.md (hunk 3 above)
Comment (bheiskell) [reply cypress/README.md:25]: This should probably be Testing locally.

File: cypress/README.md (hunk 2 above)
Comment (bheiskell) [reply cypress/README.md:42]: I was thinking that was implied, but I'll make it explicit. It's not a required file.

Comment (wrycu) [reply cypress/README.md]: This isn't a question about if it's optional, it's a question of where to create the file...

**Review by wrycu — APPROVED** [review]
LGTM

**Pull request merged**